lib/vagrant-mutate/converter/libvirt.rb

```diff
@@ -10,11 +11,20 @@ def generate_metadata
       end
 
       def generate_vagrantfile
-        <<-EOF
-        config.vm.provider :libvirt do |libvirt|
-          libvirt.disk_bus = '#{@input_box.disk_interface}'
+
+        if @force_virtio == true
+          <<-EOF
+          config.vm.provider :libvirt do |libvirt|
+            libvirt.disk_bus = '#{@output_box.disk_interface}'
+          end
+          EOF
+        else
+          <<-EOF
+          config.vm.provider :libvirt do |libvirt|
+            libvirt.disk_bus = '#{@input_box.disk_interface}'
+          end
+          EOF
         end
```

Comment:
The `if` and `else` blocks duplicate the string being returned. How about separating those like

```
if @force_virtio == true
  disk_bus = 'virtio'
else
  disk_bus = @input_box.disk_interface
end

<<-EOF
config.vm.provider :libvirt do |libvirt|
  libvirt.disk_bus = '#{disk_bus}'
end
EOF
```